nvflare/private/fed/server/server_engine.py

```diff
@@ -726,11 +726,16 @@ def check_client_resources(self, resource_reqs) -> Dict[str, Tuple[bool, str]]:
         for r in replies:
             site_name = self.get_client_name_from_token(r.client_token)
             if r.reply:
-                resp = fobs.loads(r.reply.body)
-                result[site_name] = (
-                    resp.get_header(ShareableHeader.CHECK_RESOURCE_RESULT, False),
-                    resp.get_header(ShareableHeader.RESOURCE_RESERVE_TOKEN, ""),
-                )
+                error_code = r.reply.get_header(MsgHeader.RETURN_CODE, ReturnCode.OK)
```

Comment:
This is good fix. There is a similar case (Line 584) in aux_send(). Please fix it.

Reply:
That error is also serialized so it doesn't actually cause a FOBS error. I added logging when error reply is received.